wdio/wdio.sauce.conf.js

```diff
@@ -16,26 +14,6 @@ if (process.env.BROWSER && process.env.BROWSER.includes('firefox')) {
   config.services.push('firefox-profile');
 }
 
-const baseOnPrepare = config.onPrepare;
```

Comment:
Why are we removing the check?

Reply:
Because collisions this doesn't actually prevent over loading sauce. In many cases another job would start on sauce between the time the check happens and we start a job. Need another way to do this that's on the Sauce side.

Reply:
Gotcha, thanks